sql/core/src/main/scala/org/apache/spark/sql/execution/DataSourceScanExec.scala

```diff
@@ -155,7 +155,8 @@ case class RowDataSourceScanExec(
 
     Map(
       "ReadSchema" -> requiredSchema.catalogString,
-      "PushedFilters" -> seqToString(markedFilters.toSeq)) ++
+      "PushedFilters" -> seqToString(markedFilters.toSeq),
+      "PushedPredicates" -> seqToString(pushedDownOperators.pushedPredicates.map(_.describe()))) ++
```

Comment:
I think only one will be present? We can't push both v1 and v2 filters to a source.

Reply:
OK